config/vlan.py

```diff
@@ -189,97 +231,159 @@ def vlan_member():
     pass
 
 @vlan_member.command('add')
-@click.argument('vid', metavar='<vid>', required=True, type=int)
+@click.argument('vid', metavar='<vid>', required=True)
 @click.argument('port', metavar='port', required=True)
-@click.option('-u', '--untagged', is_flag=True)
+@click.option('-u', '--untagged', is_flag=True, help="Untagged status")
+@click.option('-m', '--multiple', is_flag=True, help="Add Multiple Vlan(s) in Range or in Comma separated list")
```

Comment:
same as above - could we extend the parser to account for all valid inputs and avoid adding this flag?

Reply:
Pls see multiple_vlan_parser is already able to handle single and multiple inputs. This -u untagged flag is exisiting behavior to add untagged vlan on a port/portchannel.

Reply:
Why do we need the `--multiple` flag if the parser can already handle all possible inputs?

Reply:
Please note that --multiple flag is required as we did not add our new functionality to the existing command to keep  intact the existing VLAN behavior/command.